spec/models/state_file_w2_spec.rb

```diff
@@ -71,10 +71,10 @@
     end
 
     context "states where we don't show local boxes" do
-      let(:intake) { create :state_file_nc_intake }
+      let(:intake) { create :state_file_az_intake }
 
       it "doesn't validate local tax fields" do
-        w2.locality_nm = "asdf"
+        w2.locality_nm = "0"
```

Comment:
why did this change from nc to az? both of them do not have local boxes shown, right?